src/sciline/pipeline.py

```diff
@@ -794,3 +794,14 @@ def bind_and_call(
         if not return_tuple:
             return results[0]
         return results
+
+    def copy(self) -> Pipeline:
```

Comment:
Also implement `__copy__`?

Reply:
Sure, can do.

Reply:
Do we actually need a custom `copy` implementation? Or would this be enough with the default `__copy__`?
```python
def copy(self) -> Pipeline:
    from copy import copy
    return copy(self)
```

Reply:
Right! It's deep as far as the dicts are concerned but not the dict elements.